### ISSUE TITLE
docker support

### ISSUE THREAD

**Issue by iwa** [caris-events/tunalog#8]
would be definitely cool to have a docker support instead of having to install baremetal :)

**Comment by PichuChen** [comment]
https://github.com/caris-events/tunalog/blob/master/Dockerfile

@iwa  Is this?

**Comment by YamiOdymel** [comment]
Fixed, Thanks to @gurleensethi (#12), pushed to docker hub

```bash
$ docker run --name tunalog -p 8080:8080 yamiodymel/tunalog
```

Although the `--tls-key` and `--tls-crt` arguments aren't exposed, it shouldn't matter for now.
